src/test/java/com/fishercoder/_973Test.java

```diff
@@ -0,0 +1,26 @@
+package com.fishercoder;
+
+import com.fishercoder.solutions._973;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import static org.junit.Assert.assertArrayEquals;
+
+public class _973Test {
+    private static _973.Solution1 test;
+
+    @BeforeClass
+    public static void setUp() {
+        test = new _973.Solution1();
+    }
+
+    @Test
+    public void test1() {
+        assertArrayEquals(new int[][]{{-2,2}}, test.kClosest(new int[][]{{1,3}, {-2,2}}, 1));
```

Comment:
Nit: could you add space ` ` in between each element? Otherwise build/checkstyle will fail.
e.g. change `int[][]{{-2,2}}` to `int[][]{{-2, 2}}`

Reply:
Done